src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_x.h

```diff
@@ -253,15 +253,31 @@ namespace sirf {
 			return "ImagesReconstructor";
 		}
 
+		void set_dcm_prefix(const char* dcm_prefix)
+		{
+			dcm_prefix_ = dcm_prefix;
```

Comment:
- add a check `dcm_prefix` is not null. Although why do you want a `const char*`. I'd vastly prefer a `const std::string&` (it'll auto-convert anyway)
- add a doxygen string that it needs to be set to an empty string to disable DICOM output.